README.md

```diff
@@ -1,2 +1,111 @@
 # Capstone_project_1
 First Microverse capstone project (module 1-block 5)
+
+
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="./images/my_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Capstone Project 1 README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Deployment]
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 Capstone Project 1 <a name="about-project"></a>
+
+**Capstone Project 1** is a project that showcases my abilities to build a website with HTML, CSS and JS
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- BOOTSTRAP
+- CSS
+- JS
+
+
+### Key Features <a name="key-features"></a>
+
+- **Web page with previews of my projects**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+The user will be able to see the projects I have created and collaborated in through-out my career as a web developer.
+
+To get a local copy up and running, follow these steps.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+You can view this project live using: https://cnualartc.github.io/Capstone_project_1/
```

Comment:
- [ ] Your live demo link redirects to a page like that in the image below. Please make sure it redirects to the website you created
![image](https://user-images.githubusercontent.com/45460192/219823181-4a9f3fc3-0aff-4662-b7c4-0fd7d0a4d796.png)